jobs/utils.sh

```diff
@@ -124,16 +124,16 @@ join_windows(){
     echo "Run gerrit-git-prep on $PARAMS with zuul-site=$ZUUL_SITE zuul-ref=$ZUUL_REF zuul-change=$ZUUL_CHANGE zuul-project=$ZUUL_PROJECT"
     run_wsmancmd_with_retry 3 $PARAMS "bash C:\cinder-ci\windows\scripts\gerrit-git-prep.sh --zuul-site $ZUUL_SITE --gerrit-site $ZUUL_SITE --zuul-ref $ZUUL_REF --zuul-change $ZUUL_CHANGE --zuul-project $ZUUL_PROJECT"
     echo "Ensure service is configured with winuser=$WIN_USER and winpass=$WIN_PASS"
-    run_ps_cmd_with_retry 3 $PARAMS '"C:\cinder-ci\windows\scripts\EnsureOpenStackServices.ps1 '$WIN_USER' '$WIN_PASS' >>\\'$FIXED_IP'\openstack\logs\create-environment-'$WIN_IP'.log 2>&1"'
+    run_ps_cmd_with_retry 3 $PARAMS '"C:\cinder-ci\windows\scripts\EnsureOpenStackServices.ps1 '$WIN_USER' '$WIN_PASS' >>\\'$FIXED_IP'\openstack\logs\windows\create-environment-'$WIN_IP'.log 2>&1"'
```

Comment:
again replace that "windows". Also, when you repeat a path or anything 5-10-20 times, you put that as a variable and use the variable to reference it!